core/src/main/scala/kafka/server/LogDirFailureChannel.scala

```diff
@@ -29,7 +32,7 @@ import java.util.concurrent.{ArrayBlockingQueue, ConcurrentHashMap}
  * can take the name of the new offline log directory out of the LogDirFailureChannel and handles the log failure properly.
  *
  */
-class LogDirFailureChannel(logDirNum: Int) {
+class LogDirFailureChannel(logDirNum: Int) extends Logging {
 
   private val offlineLogDirs = new ConcurrentHashMap[String, String]
   private val logDirFailureEvent = new ArrayBlockingQueue[String](logDirNum)
```

Comment:
Not related to this PR, but it's quite confusing that a queue of log dirs is called `logDirFailureEvent`. It sounds like it contains events instead of log dirs.